datatableview/utils.py

```diff
@@ -126,8 +126,8 @@ def get_model_at_related_field(model, attr):
     except FieldDoesNotExist:
         raise
 
-    if not direct and hasattr(field, 'model'):  # Reverse relationship
-        model = field.model
+    if not direct and hasattr(field, 'related_model'):  # Reverse relationship
```

Comment:
```_meta.get_field_by_name``` returns a RelatedObject in ```field```, so ```field.model``` will refer to the parent model.  We should be using the target ```related_model```